virtcontainers/qemu.go

```diff
@@ -551,6 +551,7 @@ func (q *qemu) createSandbox(ctx context.Context, id string, networkNS NetworkNa
 	}
 
 	cpuModel := q.arch.cpuModel()
+	cpuModel += "," + q.config.CpuFeatures
```

Comment:
can you update `qemu_amd64.go` please?

https://github.com/kata-containers/runtime/blob/705713b4f9bc4d1e754871d5ef1ab5e99ea71aff/virtcontainers/qemu_amd64.go#L131-L135

Reply:
Hi @devimc  Should I  also check the conflicts e.g. the user specify "pmu=on" in nested virtualization case?

Reply:
I don't think so, it's up to the user enable/disable pmu. 
With this patch `pmu` will be off by default, so I think you should remove the next sentences

```go
if q.nestedRun { 
	cpuModel += ",pmu=off" 
} 
```

Reply:
ok, got it, thanks